src/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.Tests.csproj

```diff
@@ -8,7 +8,7 @@
     <RootNamespace>System.Runtime.InteropServices</RootNamespace>
     <AssemblyName>System.Runtime.InteropServices.Tests</AssemblyName>
     <ProjectGuid>{A824F4CD-935B-4496-A1B2-C3664936DA7B}</ProjectGuid>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker Condition="'$(TargetGroup)'==''">.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
is that not the case already in https://github.com/dotnet/corefx/blob/master/src/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.Tests.builds#L7